### ISSUE TITLE
False positive unused import

### ISSUE THREAD

**Issue by borkdude** [clj-kondo/clj-kondo#1659]
```
$ clj-kondo --lint - <<< "(import '(org.bouncycastle.crypto.digests GOST3411_2012_256Digest)) (GOST3411_2012_256Digest.)"
<stdin>:1:43: warning: Unused import GOST3411_2012_256Digest
linting took 10ms, errors: 0, warnings: 1
```

There seems to be something odd with having underscores in the class name. When they are removed, the import is no longer unused.

**Comment by mikeananev** [comment]
Cool! I suspect underscores too.
